Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/spr-test-nicGold
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/spr-test-nicGold/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/spr-test-nicGold/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/spr-test-nicGold/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"4.18.0-369.el8.x86_64"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-22.03","vm_image_conf":{"image1":"rhel86.img","image2":"rhel862.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.183.5","pf1":"ens43f0","os":"RHEL"},"pkg_generator":{"host":"10.239.182.88","pf1":"ens785f0"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4Dqouna/UALz1cds/DYDIib12lRO4R4U9QQgmUx/wyv5AAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYgEJMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAnLfKUwgAAAA==[0m[Pipeline] script
[8mha:////4JUhH4fIWh5icAEY6zBr3HRHHu0lnMalVyM2fPsu3/3sAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYgIhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD0QKVszCAAAA[0m[Pipeline] {
[8mha:////4PJRxj8hD/CuSHgtWDRwAQ67XTdBRcldvWih24U4ysBWAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCoEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDREKHeHXPvt8AHN/bm5vgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics -s -m nicGold --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nicGold_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 19 items / 15 deselected / 4 selected

npxtester/Tests/testNics/test_mtu.py 2022-04-21 22:43:14 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_Jumbo_Packet_CVL_22_43_14/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_Jumbo_Packet_CVL_22_43_14
2022-04-21 22:43:14 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-21 22:43:14 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-21 22:43:14 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-21 22:43:14 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-21 22:43:14 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:43:14 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:43:14 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 22:43:14 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:43:14 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:43:14 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-21 22:43:14 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-21 22:43:14 - sshInstance.py - [line:199] - root - INFO - start to rmmod irdma....
2022-04-21 22:43:17 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-21 22:43:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:43:22 - sshInstance.py - [line:202] - root - INFO - finished rmmod irdma....
2022-04-21 22:43:22 - sshInstance.py - [line:203] - root - INFO - start to rmmod ice....
2022-04-21 22:43:25 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-21 22:43:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:43:40 - sshInstance.py - [line:206] - root - INFO - finished rmmod ice....
2022-04-21 22:43:40 - sshInstance.py - [line:208] - root - INFO - start to modprobe ice....
2022-04-21 22:43:43 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-21 22:44:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:44:25 - sshInstance.py - [line:210] - root - INFO - finished modprobe ice....
2022-04-21 22:44:28 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-21 22:44:28 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-21 22:44:36 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 22:48:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:48:27 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-21 22:48:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:48:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-21 22:48:35 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::2775:ce0a:1912:8027  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 9  bytes 3018 (2.9 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 29  bytes 4975 (4.8 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-21 22:48:44 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Thu Apr 21 22:58:00 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled WEP decryption/encryption. (Dot11)[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled IPsec encryption/authentication.[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | What is dead may never die![0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   |                     -- Python 2[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-21 22:48:51 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*14480)],iface="ens785f0")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-21 22:48:54 - test_mtu.py - [line:91] - root - INFO - success to send package to network
2022-04-21 22:48:58 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*15500)],iface="ens785f0")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-21 22:49:01 - test_mtu.py - [line:101] - root - INFO - result is right
.2022-04-21 22:49:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 22:52:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:52:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-21 22:52:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:01 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-21 22:53:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:09 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-21 22:53:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:14 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-21 22:53:14 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-21/test_PI_Networking_Jumbo_Packet_CVL_22_43_14
2022-04-21 22:53:17 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-21 22:53:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:22 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-21 22:53:22 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-21 22:53:22 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-21 22:53:22 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22
2022-04-21 22:53:22 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-21 22:53:22 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-21 22:53:22 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:22 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:23 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:23 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-21 22:53:23 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-21 22:53:23 - sshInstance.py - [line:199] - root - INFO - start to rmmod irdma....
2022-04-21 22:53:26 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-21 22:53:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:31 - sshInstance.py - [line:202] - root - INFO - finished rmmod irdma....
2022-04-21 22:53:31 - sshInstance.py - [line:203] - root - INFO - start to rmmod ice....
2022-04-21 22:53:34 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-21 22:53:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:50 - sshInstance.py - [line:206] - root - INFO - finished rmmod ice....
2022-04-21 22:53:50 - sshInstance.py - [line:208] - root - INFO - start to modprobe ice....
2022-04-21 22:53:53 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-21 22:54:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:34 - sshInstance.py - [line:210] - root - INFO - finished modprobe ice....
2022-04-21 22:54:37 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-21 22:54:37 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-21 22:54:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 22:54:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 2000
2022-04-21 22:54:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:55:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-21 23:00:48 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 13  bytes 4371 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 72  bytes 12400 (12.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


F2022-04-21 23:00:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 23:00:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 23:01:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-21 23:01:07 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
E                                 [ 50%]
npxtester/Tests/testNics/test_network_sriov_stress.py 2022-04-21 23:01:07 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-21 23:01:07 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img]_23_01_07/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img].log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img]_23_01_07
2022-04-21 23:01:07 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img] case ============================
2022-04-21 23:01:07 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-21 23:01:07 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-21 23:01:08 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 23:01:08 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 23:01:08 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 23:01:38 - test_network_sriov_stress.py - [line:62] - root - INFO - Failed:ssh to sut2 failed,please check
E2022-04-21 23:01:38 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img] case ============================
                  [ 75%]
npxtester/Tests/testNics/test_nics.py 2022-04-21 23:01:38 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-21 23:01:38 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_gold_PI_Networking_SRIOV_CheckDevice_23_01_38/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_gold_PI_Networking_SRIOV_CheckDevice_23_01_38
2022-04-21 23:01:38 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-21 23:01:38 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-21 23:01:38 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-21 23:01:53 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 23:01:53 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 23:01:53 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 23:02:23 - test_nics.py - [line:40] - root - INFO - Case Failed:ssh to sut2 failed,please check
E2022-04-21 23:02:23 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
                                  [100%]

==================================== ERRORS ====================================
___________ ERROR at call of Test_mtu.test_PI_Networking_MTU_set_CVL ___________
self = <Tests.testNics.test_mtu.Test_mtu object at 0x7f199a78fe48>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSimple
    def test_PI_Networking_MTU_set_CVL(self):
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.sut1_cli, 'ifconfig {} mtu 1500'.\
                                      format(self.sut_conf["pf1"]))
        if stdout or stderr:
            assert 200 == 400, "set sut1 mtu is error"
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.sut2_cli, 'ifconfig {} mtu 2000'.\
                                      format(self.pkg_generator["pf1"]))
        if stdout or stderr:
            assert 200 == 400, "set sut2 mtu is error"
        self.get_mac()
        if not self.sut_mac:
            assert 200 == 400, "sut mac is none"
        cmd = "scapy\n"
>       self.cn_test_sut.run_ssh_cmd(self.sut2_session, cmd, ">>>")

npxtester/Tests/testNics/test_mtu.py:122: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
npxtester/libs/commons/sshInstance.py:67: in run_ssh_cmd
    ss.sendall(command)
/usr/local/lib/python3.6/site-packages/paramiko/channel.py:846: in sendall
    sent = self.send(s)
/usr/local/lib/python3.6/site-packages/paramiko/channel.py:801: in send
    return self._send(s, m)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <paramiko.Channel 0 (closed) -> <paramiko.Transport at 0x9a865b70 (unconnected)>>
s = 'scapy\n', m = paramiko.Message(b'^\x00\x00\x00\x00')

    def _send(self, s, m):
        size = len(s)
        self.lock.acquire()
        try:
            if self.closed:
                # this doesn't seem useful, but it is the documented behavior
                # of Socket
>               raise socket.error("Socket is closed")
E               OSError: Socket is closed

/usr/local/lib/python3.6/site-packages/paramiko/channel.py:1198: OSErrorself = <Tests.testNics.test_mtu.Test_mtu object at 0x7f199a78fe48>

    def teardown(self):
        cmd1 = "ifconfig {} mtu 1500".format(self.sut_conf["pf1"])
        cmd2 = "ifconfig {} mtu 1500".format(self.pkg_generator["pf1"])
        self.cn_test_sut.ssh_exec_cmd(self.sut1_cli, cmd1)
>       self.cn_test_sut.ssh_exec_cmd(self.sut2_cli, cmd2)

npxtester/Tests/testNics/test_mtu.py:46: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
npxtester/libs/commons/sshInstance.py:51: in ssh_exec_cmd
    stdin1, stdout1, stderr = client.exec_command(cmd, get_pty=True)
/usr/local/lib/python3.6/site-packages/paramiko/client.py:508: in exec_command
    chan = self._transport.open_session(timeout=timeout)
/usr/local/lib/python3.6/site-packages/paramiko/transport.py:879: in open_session
    timeout=timeout,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <paramiko.Transport at 0x9a865b70 (unconnected)>, kind = 'session'
dest_addr = None, src_addr = None, window_size = None, max_packet_size = None
timeout = None

    def open_channel(
        self,
        kind,
        dest_addr=None,
        src_addr=None,
        window_size=None,
        max_packet_size=None,
        timeout=None,
    ):
        """
        Request a new channel to the server. `Channels <.Channel>` are
        socket-like objects used for the actual transfer of data across the
        session. You may only request a channel after negotiating encryption
        (using `connect` or `start_client`) and authenticating.
    
        .. note:: Modifying the the window and packet sizes might have adverse
            effects on the channel created. The default values are the same
            as in the OpenSSH code base and have been battle tested.
    
        :param str kind:
            the kind of channel requested (usually ``"session"``,
            ``"forwarded-tcpip"``, ``"direct-tcpip"``, or ``"x11"``)
        :param tuple dest_addr:
            the destination address (address + port tuple) of this port
            forwarding, if ``kind`` is ``"forwarded-tcpip"`` or
            ``"direct-tcpip"`` (ignored for other channel types)
        :param src_addr: the source address of this port forwarding, if
            ``kind`` is ``"forwarded-tcpip"``, ``"direct-tcpip"``, or ``"x11"``
        :param int window_size:
            optional window size for this session.
        :param int max_packet_size:
            optional max packet size for this session.
        :param float timeout:
            optional timeout opening a channel, default 3600s (1h)
    
        :return: a new `.Channel` on success
    
        :raises:
            `.SSHException` -- if the request is rejected, the session ends
            prematurely or there is a timeout openning a channel
    
        .. versionchanged:: 1.15
            Added the ``window_size`` and ``max_packet_size`` arguments.
        """
        if not self.active:
>           raise SSHException("SSH session not active")
E           paramiko.ssh_exception.SSHException: SSH session not active

/usr/local/lib/python3.6/site-packages/paramiko/transport.py:969: SSHException
---------------------------- Captured stdout setup -----------------------------
2022-04-21 22:53:22 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22
2022-04-21 22:53:22 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-21 22:53:22 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-21 22:53:22 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:22 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:23 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:23 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-21 22:53:23 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-21 22:53:23 - sshInstance.py - [line:199] - root - INFO - start to rmmod irdma....
2022-04-21 22:53:26 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-21 22:53:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:31 - sshInstance.py - [line:202] - root - INFO - finished rmmod irdma....
2022-04-21 22:53:31 - sshInstance.py - [line:203] - root - INFO - start to rmmod ice....
2022-04-21 22:53:34 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-21 22:53:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:50 - sshInstance.py - [line:206] - root - INFO - finished rmmod ice....
2022-04-21 22:53:50 - sshInstance.py - [line:208] - root - INFO - start to modprobe ice....
2022-04-21 22:53:53 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-21 22:54:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:34 - sshInstance.py - [line:210] - root - INFO - finished modprobe ice....
2022-04-21 22:54:37 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-21 22:54:37 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

----------------------------- Captured stdout call -----------------------------
2022-04-21 22:54:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 22:54:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 2000
2022-04-21 22:54:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:55:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-21 23:00:48 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 13  bytes 4371 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 72  bytes 12400 (12.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 ifconfig ens43f0 mtu 1500
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f0 mtu 2000
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens43f0
INFO     root:sshInstance.py:54 ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 13  bytes 4371 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 72  bytes 12400 (12.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
--------------------------- Captured stdout teardown ---------------------------
2022-04-21 23:00:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 23:00:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 23:01:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-21 23:01:07 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
---------------------------- Captured log teardown -----------------------------
INFO     root:sshInstance.py:50 ifconfig ens43f0 mtu 1500
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f0 mtu 1500
INFO     root:NicTestFixture.py:35 ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
_ ERROR at setup of Test_Network_SRIOV_Stress.test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img] _

self = <Tests.testNics.test_network_sriov_stress.Test_Network_SRIOV_Stress object at 0x7f199a637630>

    def setup(self):
        logging.info("SSH SUT and setup configurations in the setup function.")
        self.sut_info = settings.__getattribute__("sut_conf")
        self.sut2_info = settings.__getattribute__("pkg_generator")
        self.dpdk_dir = settings.__getattribute__("dpdk_dir")
        self.vm_image_conf = settings.__getattribute__("vm_image_conf")
        self.vm_image_dir = self.vm_image_conf["image_dir"]
        self.pf_name = self.sut_info["pf1"]
        self.pf2_name = self.sut2_info["pf1"]
        self.network = settings.__getattribute__("network")
        self.defult_network = self.network["network_name"]
        self.defaultxml_network_path = self.network["defaultxml"]
        self.vm_info = settings.__getattribute__("vm_conf")
        self.check_str = settings.__getattribute__("check_str")
        self.drive_info = settings.__getattribute__("drive_info")
        self.vm1_name = self.vm_info["vm1_name"]
        self.vm2_name = self.vm_info["vm2_name"]
        self.run_time = self.sut_info["run_time"]
        self.ssh_sut = sshInstance()
        logging.info("ssh to the sut1")
        try:
            self.client, self.session = self.ssh_sut.get_ssh_instance(self.sut_info["host"], self.sut_info["user"],
                                                        self.sut_info["password"], self.sut_info["port"])
        except:
            logging.info("Failed to ssh to sut, please check")
            assert 200 == 400, "Failed to ssh to sut, please check"
    
        try:
            self.client2, self.session2 = self.ssh_sut.get_ssh_instance(self.sut2_info["host"], self.sut2_info["user"],
>                                                       self.sut2_info["password"],self.sut2_info["port"])

npxtester/Tests/testNics/test_network_sriov_stress.py:60: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <libs.commons.sshInstance.sshInstance object at 0x7f199a865fd0>
hostname = '10.239.182.88', username = 'root', password = 'password', port = 22

    def get_ssh_instance(self, hostname, username, password, port):
        client = paramiko.SSHClient()
        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
>       client.connect(hostname=hostname, port=port, username=username, password=password, timeout=30)

npxtester/libs/commons/sshInstance.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <paramiko.client.SSHClient object at 0x7f199a765cc0>
hostname = '10.239.182.88', port = 22, username = 'root', password = 'password'
pkey = None, key_filename = None, timeout = 30, allow_agent = True
look_for_keys = True, compress = False
sock = <socket.socket fd=12, family=AddressFamily.AF_INET, type=2049, proto=0, laddr=('10.112.97.60', 36530)>
gss_auth = False, gss_kex = False, gss_deleg_creds = True, gss_host = None
banner_timeout = None, auth_timeout = None, gss_trust_dns = True
passphrase = None, disabled_algorithms = None

    def connect(
        self,
        hostname,
        port=SSH_PORT,
        username=None,
        password=None,
        pkey=None,
        key_filename=None,
        timeout=None,
        allow_agent=True,
        look_for_keys=True,
        compress=False,
        sock=None,
        gss_auth=False,
        gss_kex=False,
        gss_deleg_creds=True,
        gss_host=None,
        banner_timeout=None,
        auth_timeout=None,
        gss_trust_dns=True,
        passphrase=None,
        disabled_algorithms=None,
    ):
        """
        Connect to an SSH server and authenticate to it.  The server's host key
        is checked against the system host keys (see `load_system_host_keys`)
        and any local host keys (`load_host_keys`).  If the server's hostname
        is not found in either set of host keys, the missing host key policy
        is used (see `set_missing_host_key_policy`).  The default policy is
        to reject the key and raise an `.SSHException`.
    
        Authentication is attempted in the following order of priority:
    
            - The ``pkey`` or ``key_filename`` passed in (if any)
    
              - ``key_filename`` may contain OpenSSH public certificate paths
                as well as regular private-key paths; when files ending in
                ``-cert.pub`` are found, they are assumed to match a private
                key, and both components will be loaded. (The private key
                itself does *not* need to be listed in ``key_filename`` for
                this to occur - *just* the certificate.)
    
            - Any key we can find through an SSH agent
            - Any "id_rsa", "id_dsa" or "id_ecdsa" key discoverable in
              ``~/.ssh/``
    
              - When OpenSSH-style public certificates exist that match an
                existing such private key (so e.g. one has ``id_rsa`` and
                ``id_rsa-cert.pub``) the certificate will be loaded alongside
                the private key and used for authentication.
    
            - Plain username/password auth, if a password was given
    
        If a private key requires a password to unlock it, and a password is
        passed in, that password will be used to attempt to unlock the key.
    
        :param str hostname: the server to connect to
        :param int port: the server port to connect to
        :param str username:
            the username to authenticate as (defaults to the current local
            username)
        :param str password:
            Used for password authentication; is also used for private key
            decryption if ``passphrase`` is not given.
        :param str passphrase:
            Used for decrypting private keys.
        :param .PKey pkey: an optional private key to use for authentication
        :param str key_filename:
            the filename, or list of filenames, of optional private key(s)
            and/or certs to try for authentication
        :param float timeout:
            an optional timeout (in seconds) for the TCP connect
        :param bool allow_agent:
            set to False to disable connecting to the SSH agent
        :param bool look_for_keys:
            set to False to disable searching for discoverable private key
            files in ``~/.ssh/``
        :param bool compress: set to True to turn on compression
        :param socket sock:
            an open socket or socket-like object (such as a `.Channel`) to use
            for communication to the target host
        :param bool gss_auth:
            ``True`` if you want to use GSS-API authentication
        :param bool gss_kex:
            Perform GSS-API Key Exchange and user authentication
        :param bool gss_deleg_creds: Delegate GSS-API client credentials or not
        :param str gss_host:
            The targets name in the kerberos database. default: hostname
        :param bool gss_trust_dns:
            Indicates whether or not the DNS is trusted to securely
            canonicalize the name of the host being connected to (default
            ``True``).
        :param float banner_timeout: an optional timeout (in seconds) to wait
            for the SSH banner to be presented.
        :param float auth_timeout: an optional timeout (in seconds) to wait for
            an authentication response.
        :param dict disabled_algorithms:
            an optional dict passed directly to `.Transport` and its keyword
            argument of the same name.
    
        :raises:
            `.BadHostKeyException` -- if the server's host key could not be
            verified
        :raises: `.AuthenticationException` -- if authentication failed
        :raises:
            `.SSHException` -- if there was any other error connecting or
            establishing an SSH session
        :raises socket.error: if a socket error occurred while connecting
    
        .. versionchanged:: 1.15
            Added the ``banner_timeout``, ``gss_auth``, ``gss_kex``,
            ``gss_deleg_creds`` and ``gss_host`` arguments.
        .. versionchanged:: 2.3
            Added the ``gss_trust_dns`` argument.
        .. versionchanged:: 2.4
            Added the ``passphrase`` argument.
        .. versionchanged:: 2.6
            Added the ``disabled_algorithms`` argument.
        """
        if not sock:
            errors = {}
            # Try multiple possible address families (e.g. IPv4 vs IPv6)
            to_try = list(self._families_and_addresses(hostname, port))
            for af, addr in to_try:
                try:
                    sock = socket.socket(af, socket.SOCK_STREAM)
                    if timeout is not None:
                        try:
                            sock.settimeout(timeout)
                        except:
                            pass
>                   retry_on_signal(lambda: sock.connect(addr))

/usr/local/lib/python3.6/site-packages/paramiko/client.py:349: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

function = <function SSHClient.connect.<locals>.<lambda> at 0x7f199a629158>

    def retry_on_signal(function):
        """Retries function until it doesn't raise an EINTR error"""
        while True:
            try:
>               return function()

/usr/local/lib/python3.6/site-packages/paramiko/util.py:283: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   retry_on_signal(lambda: sock.connect(addr))
E   socket.timeout: timed out

/usr/local/lib/python3.6/site-packages/paramiko/client.py:349: timeout

During handling of the above exception, another exception occurred:

self = <Tests.testNics.test_network_sriov_stress.Test_Network_SRIOV_Stress object at 0x7f199a637630>

    def setup(self):
        logging.info("SSH SUT and setup configurations in the setup function.")
        self.sut_info = settings.__getattribute__("sut_conf")
        self.sut2_info = settings.__getattribute__("pkg_generator")
        self.dpdk_dir = settings.__getattribute__("dpdk_dir")
        self.vm_image_conf = settings.__getattribute__("vm_image_conf")
        self.vm_image_dir = self.vm_image_conf["image_dir"]
        self.pf_name = self.sut_info["pf1"]
        self.pf2_name = self.sut2_info["pf1"]
        self.network = settings.__getattribute__("network")
        self.defult_network = self.network["network_name"]
        self.defaultxml_network_path = self.network["defaultxml"]
        self.vm_info = settings.__getattribute__("vm_conf")
        self.check_str = settings.__getattribute__("check_str")
        self.drive_info = settings.__getattribute__("drive_info")
        self.vm1_name = self.vm_info["vm1_name"]
        self.vm2_name = self.vm_info["vm2_name"]
        self.run_time = self.sut_info["run_time"]
        self.ssh_sut = sshInstance()
        logging.info("ssh to the sut1")
        try:
            self.client, self.session = self.ssh_sut.get_ssh_instance(self.sut_info["host"], self.sut_info["user"],
                                                        self.sut_info["password"], self.sut_info["port"])
        except:
            logging.info("Failed to ssh to sut, please check")
            assert 200 == 400, "Failed to ssh to sut, please check"
    
        try:
            self.client2, self.session2 = self.ssh_sut.get_ssh_instance(self.sut2_info["host"], self.sut2_info["user"],
                                                        self.sut2_info["password"],self.sut2_info["port"])
        except:
            logging.info("Failed:ssh to sut2 failed,please check")
>           assert 200 == 400, "Failed:ssh to sut2 failed,please check"
E           AssertionError: Failed:ssh to sut2 failed,please check
E           assert 200 == 400

npxtester/Tests/testNics/test_network_sriov_stress.py:63: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-21 23:01:07 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-21 23:01:07 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img]_23_01_07/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img].log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img]_23_01_07
2022-04-21 23:01:07 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img] case ============================
2022-04-21 23:01:07 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-21 23:01:07 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-21 23:01:08 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 23:01:08 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 23:01:08 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 23:01:38 - test_network_sriov_stress.py - [line:62] - root - INFO - Failed:ssh to sut2 failed,please check
------------------------------ Captured log setup ------------------------------
INFO     root:NicTestFixture.py:20 test -------
____ ERROR at setup of Test_Nics.test_gold_PI_Networking_SRIOV_CheckDevice _____

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f199a665ac8>

    def setup(self):
        logging.info("this is Test_Nics setup.")
        self.sut_conf = settings.__getattribute__("sut_conf")
        self.pkg_generator = settings.__getattribute__("pkg_generator")
        self.drive_info = settings.__getattribute__("drive_info")
        self.check_str = settings.__getattribute__("check_str")
        self.dpdk_dir = settings.__getattribute__("dpdk_dir")
        self.cn_test_sut = sshInstance()
        logging.info("ssh to the sut1")
        try:
            self.client, self.session = self.cn_test_sut.get_ssh_instance(self.sut_conf["host"], self.sut_conf["user"],
                                                        self.sut_conf["password"], self.sut_conf["port"])
        except:
            logging.info("Case Failed:ssh to sut failed,please check")
            assert 200 == 400, "Case Failed:ssh to sut failed,please check"
    
        try:
            self.client2, self.session2 = self.cn_test_sut.get_ssh_instance(self.pkg_generator["host"], self.pkg_generator["user"],
>                                                       self.pkg_generator["password"], self.pkg_generator["port"])

npxtester/Tests/testNics/test_nics.py:38: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <libs.commons.sshInstance.sshInstance object at 0x7f199a865630>
hostname = '10.239.182.88', username = 'root', password = 'password', port = 22

    def get_ssh_instance(self, hostname, username, password, port):
        client = paramiko.SSHClient()
        client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
>       client.connect(hostname=hostname, port=port, username=username, password=password, timeout=30)

npxtester/libs/commons/sshInstance.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <paramiko.client.SSHClient object at 0x7f199a579390>
hostname = '10.239.182.88', port = 22, username = 'root', password = 'password'
pkey = None, key_filename = None, timeout = 30, allow_agent = True
look_for_keys = True, compress = False
sock = <socket.socket fd=15, family=AddressFamily.AF_INET, type=2049, proto=0, laddr=('10.112.97.60', 36534)>
gss_auth = False, gss_kex = False, gss_deleg_creds = True, gss_host = None
banner_timeout = None, auth_timeout = None, gss_trust_dns = True
passphrase = None, disabled_algorithms = None

    def connect(
        self,
        hostname,
        port=SSH_PORT,
        username=None,
        password=None,
        pkey=None,
        key_filename=None,
        timeout=None,
        allow_agent=True,
        look_for_keys=True,
        compress=False,
        sock=None,
        gss_auth=False,
        gss_kex=False,
        gss_deleg_creds=True,
        gss_host=None,
        banner_timeout=None,
        auth_timeout=None,
        gss_trust_dns=True,
        passphrase=None,
        disabled_algorithms=None,
    ):
        """
        Connect to an SSH server and authenticate to it.  The server's host key
        is checked against the system host keys (see `load_system_host_keys`)
        and any local host keys (`load_host_keys`).  If the server's hostname
        is not found in either set of host keys, the missing host key policy
        is used (see `set_missing_host_key_policy`).  The default policy is
        to reject the key and raise an `.SSHException`.
    
        Authentication is attempted in the following order of priority:
    
            - The ``pkey`` or ``key_filename`` passed in (if any)
    
              - ``key_filename`` may contain OpenSSH public certificate paths
                as well as regular private-key paths; when files ending in
                ``-cert.pub`` are found, they are assumed to match a private
                key, and both components will be loaded. (The private key
                itself does *not* need to be listed in ``key_filename`` for
                this to occur - *just* the certificate.)
    
            - Any key we can find through an SSH agent
            - Any "id_rsa", "id_dsa" or "id_ecdsa" key discoverable in
              ``~/.ssh/``
    
              - When OpenSSH-style public certificates exist that match an
                existing such private key (so e.g. one has ``id_rsa`` and
                ``id_rsa-cert.pub``) the certificate will be loaded alongside
                the private key and used for authentication.
    
            - Plain username/password auth, if a password was given
    
        If a private key requires a password to unlock it, and a password is
        passed in, that password will be used to attempt to unlock the key.
    
        :param str hostname: the server to connect to
        :param int port: the server port to connect to
        :param str username:
            the username to authenticate as (defaults to the current local
            username)
        :param str password:
            Used for password authentication; is also used for private key
            decryption if ``passphrase`` is not given.
        :param str passphrase:
            Used for decrypting private keys.
        :param .PKey pkey: an optional private key to use for authentication
        :param str key_filename:
            the filename, or list of filenames, of optional private key(s)
            and/or certs to try for authentication
        :param float timeout:
            an optional timeout (in seconds) for the TCP connect
        :param bool allow_agent:
            set to False to disable connecting to the SSH agent
        :param bool look_for_keys:
            set to False to disable searching for discoverable private key
            files in ``~/.ssh/``
        :param bool compress: set to True to turn on compression
        :param socket sock:
            an open socket or socket-like object (such as a `.Channel`) to use
            for communication to the target host
        :param bool gss_auth:
            ``True`` if you want to use GSS-API authentication
        :param bool gss_kex:
            Perform GSS-API Key Exchange and user authentication
        :param bool gss_deleg_creds: Delegate GSS-API client credentials or not
        :param str gss_host:
            The targets name in the kerberos database. default: hostname
        :param bool gss_trust_dns:
            Indicates whether or not the DNS is trusted to securely
            canonicalize the name of the host being connected to (default
            ``True``).
        :param float banner_timeout: an optional timeout (in seconds) to wait
            for the SSH banner to be presented.
        :param float auth_timeout: an optional timeout (in seconds) to wait for
            an authentication response.
        :param dict disabled_algorithms:
            an optional dict passed directly to `.Transport` and its keyword
            argument of the same name.
    
        :raises:
            `.BadHostKeyException` -- if the server's host key could not be
            verified
        :raises: `.AuthenticationException` -- if authentication failed
        :raises:
            `.SSHException` -- if there was any other error connecting or
            establishing an SSH session
        :raises socket.error: if a socket error occurred while connecting
    
        .. versionchanged:: 1.15
            Added the ``banner_timeout``, ``gss_auth``, ``gss_kex``,
            ``gss_deleg_creds`` and ``gss_host`` arguments.
        .. versionchanged:: 2.3
            Added the ``gss_trust_dns`` argument.
        .. versionchanged:: 2.4
            Added the ``passphrase`` argument.
        .. versionchanged:: 2.6
            Added the ``disabled_algorithms`` argument.
        """
        if not sock:
            errors = {}
            # Try multiple possible address families (e.g. IPv4 vs IPv6)
            to_try = list(self._families_and_addresses(hostname, port))
            for af, addr in to_try:
                try:
                    sock = socket.socket(af, socket.SOCK_STREAM)
                    if timeout is not None:
                        try:
                            sock.settimeout(timeout)
                        except:
                            pass
>                   retry_on_signal(lambda: sock.connect(addr))

/usr/local/lib/python3.6/site-packages/paramiko/client.py:349: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

function = <function SSHClient.connect.<locals>.<lambda> at 0x7f199a71bbf8>

    def retry_on_signal(function):
        """Retries function until it doesn't raise an EINTR error"""
        while True:
            try:
>               return function()

/usr/local/lib/python3.6/site-packages/paramiko/util.py:283: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   retry_on_signal(lambda: sock.connect(addr))
E   socket.timeout: timed out

/usr/local/lib/python3.6/site-packages/paramiko/client.py:349: timeout

During handling of the above exception, another exception occurred:

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f199a665ac8>

    def setup(self):
        logging.info("this is Test_Nics setup.")
        self.sut_conf = settings.__getattribute__("sut_conf")
        self.pkg_generator = settings.__getattribute__("pkg_generator")
        self.drive_info = settings.__getattribute__("drive_info")
        self.check_str = settings.__getattribute__("check_str")
        self.dpdk_dir = settings.__getattribute__("dpdk_dir")
        self.cn_test_sut = sshInstance()
        logging.info("ssh to the sut1")
        try:
            self.client, self.session = self.cn_test_sut.get_ssh_instance(self.sut_conf["host"], self.sut_conf["user"],
                                                        self.sut_conf["password"], self.sut_conf["port"])
        except:
            logging.info("Case Failed:ssh to sut failed,please check")
            assert 200 == 400, "Case Failed:ssh to sut failed,please check"
    
        try:
            self.client2, self.session2 = self.cn_test_sut.get_ssh_instance(self.pkg_generator["host"], self.pkg_generator["user"],
                                                        self.pkg_generator["password"], self.pkg_generator["port"])
        except:
            logging.info("Case Failed:ssh to sut2 failed,please check")
>           assert 200 == 400, "Case Failed:ssh to sut2 failed,please check"
E           AssertionError: Case Failed:ssh to sut2 failed,please check
E           assert 200 == 400

npxtester/Tests/testNics/test_nics.py:41: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-21 23:01:38 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-21 23:01:38 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_gold_PI_Networking_SRIOV_CheckDevice_23_01_38/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_gold_PI_Networking_SRIOV_CheckDevice_23_01_38
2022-04-21 23:01:38 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-21 23:01:38 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-21 23:01:38 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-21 23:01:53 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 23:01:53 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 23:01:53 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 23:02:23 - test_nics.py - [line:40] - root - INFO - Case Failed:ssh to sut2 failed,please check
------------------------------ Captured log setup ------------------------------
INFO     root:NicTestFixture.py:20 test -------
=================================== FAILURES ===================================
___________________ Test_mtu.test_PI_Networking_MTU_set_CVL ____________________

self = <Tests.testNics.test_mtu.Test_mtu object at 0x7f199a78fe48>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSimple
    def test_PI_Networking_MTU_set_CVL(self):
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.sut1_cli, 'ifconfig {} mtu 1500'.\
                                      format(self.sut_conf["pf1"]))
        if stdout or stderr:
            assert 200 == 400, "set sut1 mtu is error"
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.sut2_cli, 'ifconfig {} mtu 2000'.\
                                      format(self.pkg_generator["pf1"]))
        if stdout or stderr:
            assert 200 == 400, "set sut2 mtu is error"
        self.get_mac()
        if not self.sut_mac:
            assert 200 == 400, "sut mac is none"
        cmd = "scapy\n"
>       self.cn_test_sut.run_ssh_cmd(self.sut2_session, cmd, ">>>")

npxtester/Tests/testNics/test_mtu.py:122: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
npxtester/libs/commons/sshInstance.py:67: in run_ssh_cmd
    ss.sendall(command)
/usr/local/lib/python3.6/site-packages/paramiko/channel.py:846: in sendall
    sent = self.send(s)
/usr/local/lib/python3.6/site-packages/paramiko/channel.py:801: in send
    return self._send(s, m)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <paramiko.Channel 0 (closed) -> <paramiko.Transport at 0x9a865b70 (unconnected)>>
s = 'scapy\n', m = paramiko.Message(b'^\x00\x00\x00\x00')

    def _send(self, s, m):
        size = len(s)
        self.lock.acquire()
        try:
            if self.closed:
                # this doesn't seem useful, but it is the documented behavior
                # of Socket
>               raise socket.error("Socket is closed")
E               OSError: Socket is closed

/usr/local/lib/python3.6/site-packages/paramiko/channel.py:1198: OSError
---------------------------- Captured stdout setup -----------------------------
2022-04-21 22:53:22 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-21/test_PI_Networking_MTU_set_CVL_22_53_22
2022-04-21 22:53:22 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-21 22:53:22 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-21 22:53:22 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:22 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.183.5
2022-04-21 22:53:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-21 22:53:23 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-21 22:53:23 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-21 22:53:23 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-21 22:53:23 - sshInstance.py - [line:199] - root - INFO - start to rmmod irdma....
2022-04-21 22:53:26 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-21 22:53:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:31 - sshInstance.py - [line:202] - root - INFO - finished rmmod irdma....
2022-04-21 22:53:31 - sshInstance.py - [line:203] - root - INFO - start to rmmod ice....
2022-04-21 22:53:34 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-21 22:53:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:53:50 - sshInstance.py - [line:206] - root - INFO - finished rmmod ice....
2022-04-21 22:53:50 - sshInstance.py - [line:208] - root - INFO - start to modprobe ice....
2022-04-21 22:53:53 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-21 22:54:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:34 - sshInstance.py - [line:210] - root - INFO - finished modprobe ice....
2022-04-21 22:54:37 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-21 22:54:37 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

----------------------------- Captured stdout call -----------------------------
2022-04-21 22:54:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-21 22:54:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:54:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 2000
2022-04-21 22:54:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-21 22:55:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-21 23:00:48 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 13  bytes 4371 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 72  bytes 12400 (12.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 ifconfig ens43f0 mtu 1500
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f0 mtu 2000
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens43f0
INFO     root:sshInstance.py:54 ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 13  bytes 4371 (4.2 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 72  bytes 12400 (12.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:90
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_iperf_client_4hour.py:90: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:143
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_iperf_client_4hour.py:143: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:108
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:108: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:592
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:592: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:719
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:719: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:959
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:959: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1095
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:1095: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1096
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:1096: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:93
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:93: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:132
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:132: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:167
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:167: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:168
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:168: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:256
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:256: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:282
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:282: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:325
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:325: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nicGold_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_mtu.py::Test_mtu::test_PI_Networking_MTU_set_CVL
FAILED npxtester/Tests/testNics/test_mtu.py::Test_mtu::test_PI_Networking_MTU_set_CVL
ERROR npxtester/Tests/testNics/test_network_sriov_stress.py::Test_Network_SRIOV_Stress::test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel862.img-rhel86.img]
ERROR npxtester/Tests/testNics/test_nics.py::Test_Nics::test_gold_PI_Networking_SRIOV_CheckDevice
= 1 failed, 1 passed, 15 deselected, 16 warnings, 3 errors in 1166.52s (0:19:26) =
[8mha:////4DDaF6xD1n97fCJC17HEPtC4MvFSxjBMoUPdpyGs71EZAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIioEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDREKHeHXPvt8AGMzKKgvgAAAA==[0m[Pipeline] echo
1
[8mha:////4JdwCG2vyNU1vR63Cq0h1I9fweuZdBzoUYFbNZMa07ZpAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIihQohqrRWGl5gYmOcWHfGdkgqXsTX+AMWkajYYrWzzbzesI4BThwM9poGS7Gz6N1o8sKJw3B1PGHPF+yYIjuNrZ5aVrpZsOWkYUlRwkrARlPnOFoyCbailw9ZOUmmOqeQv6OA0qo7PKEQ2ZxkSF+YA+xuo4pMP89fAcw+QVkfcu1r/wFPn4+LvgAAAA==[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4J0lAWMm8HuBR5s3oE3lokuIt9KB2+AHQUkjEiJ+aQGpAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPMKRSIUQVVorDS8w8WGcWHfGdkgqXsTX+AMWkajYYrWzzbzesI4BThyM6JEGS7GzwrvR5CUmDsPV8SR6voiOKbJD0eLUssZmwZYTwpKihJWEDVLnOFoyCbayVw9VOUWmOqeQv6OE0uo7PKGQ2ZxUSF+YA+xuo45MP89fAcw+Qbk/5Kpr/wEOrpSSvgAAAA==[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4Mt0DSsxkJzZfVcF7hGhEOllvDy/ttdaA/Rf0YiZvcahAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAQ1CqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKYhQrk95Nrthw/JONXdvgAAAA==[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4BJ3niM699yfMaO4XlYckMNVxHBxF/A/6tXrT01aXGOTAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigSgQSkVrueEFJjHGiXUXbAen4kV8jT9gEYmKLVY728zrDcvgoWZvsNPUWwqNxcGNJi9M7Pur44QdX7BhCuw0Sp0kt/o0o+SoYU5RwkLASlPjOFgyEdaiUw9VOUWmOkefv6OA0rZ3eEIhsjkqH78wedjcxjYw/Tx/BTANQ4Ryt8+1PXwARMxJg74AAAA=[0m[Pipeline] }
[8mha:////4KLOGR+83vgrLFidGueNeEu4h+87+yS5qW1q0kigPKEkAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigYSQUCpayw0vMIkxTqy7YDs4FS/ia/wBi0hUbLHa2WZeb1gGDzV7g52m3lJoLA5uNHlhYt9fHSfs+IINU2CnUeokudWnGSVHDXOKEhYCVpoax8GSibAWnXqoyiky1Tn6/B0FlLa9wxMKkc1R+fiFycPmNraB6ef5K4BpGCKUu32u7eEDfI7acr4AAAA=[0m[Pipeline] // script
[8mha:////4B/bjeg5xruNbbx3kITLupXWvVAJcKllvbrOrw34K5MTAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiQZECUaW10vACExvjxLoztkNS8SK+xh+wiETFFqudbeb1hnUMcOJgsNc0WIqdRe9GkxdOHIar4wl7vmDHFNlpbPXUstLNgi0nDUuKElYCNpo6x9GSSbAVvXzIykky1TmF/B0FlFbd4QmFyOYkQ/rCHGB3G1Vk+nn+CmD2PkF5qHPt6w9IQDkpvgAAAA==[0m[Pipeline] }
[8mha:////4Pysn2p6Cuaw38A5WKCzX3hZbaQrVsudvseeZZRTcVjBAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigYSQUCpayw0vMIkxTqy7YDs4FS/ia/wBi0hUbLHa2WZeb1gGDzV7g52m3lJoLA5uNHlhYt9fHSfs+IINU2CnUeokudWnGSVHDXOKEhYCVpoax8GSibAWnXqoyiky1Tn6/B0FlLa9wxMKkc1R+fiFycPmNraB6ef5K4BpGCKUu0Ou7f4DC8Jor74AAAA=[0m[Pipeline] // stage
[8mha:////4AtNwyaW/AZqBd3OKxZrkWfAC6zB3HxngMFQ9sJmqFs1AAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEGIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsP/+idkr0AAAA=[0m[Pipeline] }
[8mha:////4I5dJP3UyABwbWbQZBRGWAtgfXC/GfHsQ3SW1rlV0+rIAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBgg5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBlS/sHL0AAAA=[0m[Pipeline] // node
[8mha:////4OCxxHdXTRYePL6pNSOSjI/nVmiKF6Ardumg4UkWHzykAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBszaLVr0AAAA=[0m[Pipeline] End of Pipeline
Finished: FAILURE
